src/ResourceManager/Network/Commands.Network/Commands.Network.csproj

```diff
@@ -58,7 +58,8 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
bump this version as well to 20.1.0

Also, update the [netcore csproj](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/Commands.Network.Netcore.csproj) as well

Reply:
Done